dev-python/simplejson/simplejson-3.19.1.ebuild

```diff
@@ -25,6 +25,7 @@ DOCS=( README.rst CHANGES.txt )
 distutils_enable_tests unittest
 
 src_configure() {
+	# native-extensions not build on PYPY
```

Comment:
Perhaps explain that the vars below are ignored.

Reply:
Added urls for logic which disables it for pypy or in general for all non-cython.